GettingStarted.adoc

```diff
@@ -1342,14 +1342,14 @@ In `src/dev/user.clj`, we'll add the following for development use:
 (defn go
   "Start the server. Optionally supply a path to your desired config. Relative paths will scan classpath. Absolute
   paths will come from the filesystem. The default is config/dev.edn."
-  ([] (go :dev))
+  ([] (go "config/dev.edn))
```

Comment:
missing a quote?

Reply:
Yep, a PR after a few drinks will do that to you 😅